wordpress-mobile/WordPress-Android
# [Tags Feed] Implement Post Item UI improvements

Fixes #20788 

Implement UI improvements to the post item in Tags Feed, making it more compact.

-----

## To Test:

1. Open Jetpack
2. Make sure the `reader_tags_feed` Feature Config is ON
3. Go to Reader
4. Select `Tags` feed
5. **Verify** the post items have the new design

-----

## Regression Notes

1. Potential unintended areas of impact

    - N/A

6. What I did to test those areas of impact (or what existing automated tests I relied on)

    - N/A

7. What automated tests I added (or what prevented me from doing so)

    - N/A, UI changes only

-----

## PR Submission Checklist:


-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):
